examples/server.rs

```diff
@@ -0,0 +1,15 @@
+use twitch_oauth2_auth_flow::auth_flow;
+
+#[tokio::main]
+async fn main() {
+    let _ = dotenv::dotenv();
+    let client_id = get_var("CLIENT_ID");
+    let client_secret = get_var("CLIENT_SECRET");
+    let scopes = None;
+    let res = auth_flow(&client_id, &client_secret, scopes, "http://localhost");
```

Comment:
I would leave http://localhost:SOMEPORT in the example. port 80 requires root so I wouldn't expect anyone to use lib in a root process.